translations/pt-BR/README.md

```diff
@@ -0,0 +1,381 @@
+    "description": "Colaboração para criar um artigo amigável para iniciantes para ajudar a explicar os Static Web Apps do Azure e as ferramentas para começar.",
+    "url": "https://dev.to/azure/10-things-to-know-about-azure-static-web-apps-3n4i"
+  },
+  {
+    "title": "Desenvolvimento Web para Iniciantes",
+    "description": "Contribuiu com imagens de notas de esboço para acompanhar cada lição. Estes ajudam a fornecer representação visual do que está sendo ensinado.",
+    "url": "https://github.com/microsoft/web-dev-for-beginners"
+  },
+  {
+    "title": "Meu Site de Currículo",
+    "description": "Criado a partir do workshop de currículo da Microsoft e implantado em GitHub Pages. Inclui minha experiência e habilidades de design.",
+    "url": "https://github.com/microsoft/workshop-library/tree/main/full/build-resume-website"
+  },
+  {
+    "title": "GitHub Codespaces e github.dev",
+    "description": "Entrevista em vídeo para explicar quando usar GitHub.dev versus GitHub Codespaces e qual a melhor forma de usar cada ferramenta.",
+    "url": "https://www.youtube.com/watch?v=c3hHhRME_XI"
+  }
+]
+```
+
+<br/>
+
+## 🏃 Publique seu aplicativo Web
+
+Project inclui a configuração necessária para você publicar **grátis** em ambos lugares, no [Azure Static Web Apps](https://azure.microsoft.com/products/app-service/static/?WT.mc_id=dotnet-82024-juyoo) e no [GitHub Pages](https://pages.github.com/)</a>.
+
+### Azure Static Web Apps
+
+[Azure Static Web Apps](https://azure.microsoft.com/products/app-service/static/?WT.mc_id=dotnet-82024-juyoo) é a solução de hospedagem da Microsoft para sites estáticos (ou sites que são renderizados no navegador do usuário, não em um servidor) por meio do Azure. Esse serviço oferece oportunidades adicionais para expandir seu site por meio do Azure Functions, autenticação, versões de preparo e muito mais.
+
+Você precisará de contas do Azure e do GitHub para implantar seu aplicativo Web. Se você ainda não tiver uma conta do Azure, poderá criá-la de dentro durante o processo de implantação ou nos links abaixo:
+
+* [Criar uma (Cartao de Crédito nao necessário) conta do Azure For Students](https://azure.microsoft.com/free/students/?WT.mc_id=dotnet-82024-juyoo)
+* [Criar uma nova conta Azure](https://azure.microsoft.com/free/?WT.mc_id=dotnet-82024-juyoo)
+
+Com seu projeto aberto no Codespaces:
+
+1. Clique no ícone do Azure na barra lateral esquerda. Faça logon se ainda não estiver e, se for novo no Azure, siga os prompts para criar sua conta.
+2. No menu do Azure, clique em "➕" assinar e, em seguida, escolher "Create Static Web App".
+
+   <img src="../../images/deploy-to-azure.png" alt="Create Static Web App" style="width: 300px;" />
+
+3. Se você não estiver logado no GitHub, você será solicitado a fazer login. Se você tiver alguma alteração de arquivo pendente, você será solicitado a confirmar essas alterações.
+4. Defina as informações do aplicativo quando solicitado:
```

Comment:
Os nomes das definições, você pode manter em inglês (Name for Static Web App, Region, etc)